prettier.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  printWidth: 120,
+  singleQuote: true,
+  semi: false,
+  excludeFiles: 'fixtures/**/*-expected.js',
```

Comment:
`excludeFiles` is only allowed in `overrides` sections.

> [Configuration Overrides](https://prettier.io/docs/en/configuration.html#configuration-overrides) files is required for each override, and may be a string or array of strings. excludeFiles may be optionally provided to exclude files for a given rule, and may also be a string or array of strings.

Reply:
😢 do u know how to setup this correctly then? we need to have those files untouched by prettier and I find it easier to do this in overrides-like section of the config than in a single glob matching which files have to be prettified (like before)

Reply:
Oh, I find that we could [add a .prettierignore file](https://prettier.io/docs/en/ignore.html#ignoring-files)

> To exclude files from formatting, add entries to a .prettierignore file in the project root or set the --ignore-path CLI option.